src/gui/state/game/game.cpp

```diff
@@ -720,7 +720,20 @@ std::vector<Message> Game::GetMessages() const {
 
   static constexpr size_t SAFE_MAX_ACTIVE_NORMAL_PLUGINS = 254;
 
-  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS &&
+  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS && settings_.Type() == GameType::tes3 && 
+      std::filesystem::exists(preludePath_/"MWSE.dll")) {
+      auto logger = getLogger();
+    if (logger) {
+      logger->warn(
+          "More than 255 normal plugins are activated at the same time.");
```

Comment:
I just spotted that the message says more than 255 but the check is for more than 254.